cli/src/pcluster/utils.py

```diff
@@ -298,7 +318,8 @@ def timestamp_to_isoformat(timestamp, timezone=None):
     if not timezone:
         timezone = tz.tzlocal()
     # Forcing microsecond to 0 to avoid having them displayed.
-    return datetime.fromtimestamp(timestamp / 1000, tz=timezone).isoformat(timespec="seconds")
+    time_ = datetime.datetime.fromtimestamp(timestamp / 1000, tz=timezone)
+    return time_.isoformat(timespec="seconds")
```

Comment:
Could you explain this change?

Reply:
I needed access to the `datetime.date` member of the `datetime` module. Previously we were using `from datetime import datetime` which I wanted to convert to `import datetime`... so when I made this change we ended up with the `datetime.datetime.fromtimestamp(...)` call, which made the line pretty long so I decided to break it up from a chain of calls so the line length would be shorter.